<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>907</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AnhThi-HoaiThuong-PhuocThuan</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="iframe_old">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>400</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="iframe_new">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>140</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>BEFORE</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAfter">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>110</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>AFTER</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMean">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Mean</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="boxFilter">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>3</number>
    </property>
    <property name="singleStep">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Size of filter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBlur">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Blur</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGauss">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Gauss</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMedian">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Median</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGx">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Gx</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGy">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Gy</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGxAddGy">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>|Gx|+|Gy|</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGxDivGy">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Gx/Gy</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Segmentation">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Segmentation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="THRESH_GAUSSIAN_C">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>40</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>THRESH_GAUSSIAN_C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BINARY">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>BINARY</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TOZERO">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>40</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>TOZERO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TRUNC">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>TRUNC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OTSU">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>40</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>OTSU</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>907</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Files</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
